base_classes/NXbeam.nxdl.xml

```diff
@@ -61,11 +61,45 @@
         <doc>Distance from sample. Note, it is recommended to use NXtransformations instead.</doc>
     </field>
     <field name="incident_energy" type="NX_FLOAT" units="NX_ENERGY">
-        <doc>Energy carried by each particle of the beam on entering the beamline component</doc>
+        <doc>
+            Energy carried by each particle of the beam on entering the beamline component.
```

Comment:
```suggestion
            Energy carried by each particle of the beam on entering the beamline location.
```

NXbeam defines the properties of the beam at a specific location.

Reply:
Agreed, has been changed.